internal/controller/supervisorconfig/oidcupstreamwatcher/oidc_upstream_watcher.go

```diff
@@ -263,7 +263,12 @@ func (c *oidcWatcherController) validateIssuer(ctx context.Context, upstream *v1
 				Message: err.Error(),
 			}
 		}
-		httpClient = &http.Client{Transport: &http.Transport{TLSClientConfig: tlsConfig}}
+		httpClient = &http.Client{
```

Comment:
We really should move this stuff to the Kube transport code so that it can be logged at higher levels for debugging.

Reply:
Can you elaborate on this? Not sure what I need to change this to.

Reply:
@enj Is this an example of what you're referring to? 
https://github.com/vmware-tanzu/pinniped/blob/14b8fcc4723b78e7c818e616602fe2c920b1f09e/cmd/pinniped/cmd/login_oidc.go#L264-L274

Reply:
Yeah, I think that's what @enj meant. It doesn't need to be part of this PR. In fact, we have an existing issue filed here: https://github.com/vmware-tanzu/pinniped/issues/324.